Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
Speaking of which:

- https://github.com/sandstorm/TypoScriptDebugger/pull/2
- https://github.com/kitsunet/flowpack-typoscript-browser/pull/2

:D